internal/testrunner/coveragereport_test.go

```diff
@@ -212,49 +79,14 @@ func TestCreateCoverageReport(t *testing.T) {
 						},
 					},
 				},
-				TestType:  "Coverage for system test",
-				Timestamp: 20,
-			},
-		},
-		{
-			name:           "generic coverage for an input package",
-			testType:       "asset",
-			rootPath:       packageRootPath,
-			packageName:    "mypackage",
-			packageType:    "input",
-			coverageFormat: "generic",
-			timestamp:      10,
-			results: []TestResult{
-				{
-					Name:        "test1",
-					Package:     "mypackage",
-					DataStream:  "",
-					TimeElapsed: 1 * time.Second,
-					Coverage:    nil,
-				},
-			},
-			expected: &GenericCoverage{
-				Version: 1,
-				Files: []*GenericFile{
-					{
-						Path: filepath.Join("internal", "testrunner", "my", "path", "mypackage", "manifest.yml"),
-						Lines: []*GenericLine{
-							{
-								LineNumber: 1,
-								Covered:    true,
-							},
-						},
-					},
-				},
-				TestType:  "Coverage for asset test",
-				Timestamp: 10,
+				TestType: "Coverage for system test",
 			},
 		},
 	}
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			report, err := createCoverageReport(tt.rootPath, tt.packageName, tt.packageType, tt.testType, tt.results, tt.coverageFormat, tt.timestamp)
```

Comment:
Is timestamp not mandatory for the coverage reports using Cobertura?

Reply:
Maybe, but there is no coverage report generated by this method now, it uses the coverage in the results, and then the timestamp is not used anymore, this is why I removed the parameter.